app/assets/stylesheets/govuk_publishing_components/components/govspeak/_typography.scss

```diff
@@ -27,26 +27,26 @@
   // Block quotes
 
   blockquote {
-    padding: 0 0 0 $gutter-two-thirds;
+    padding: 0 0 0 govuk-spacing(4);
     margin: 0;
     border: 0;
 
-    @include media(desktop) {
-      margin: 0 0 0 (-$gutter);
+    @include govuk-media-query($from: desktop) {
+      margin: 0 0 0 (- govuk-spacing(6));
     }
 
     p {
-      padding-left: $gutter-half;
+      padding-left: govuk-spacing(3);
 
-      @include media(tablet) {
-        padding-left: $gutter;
+      @include govuk-media-query($from: tablet) {
+        padding-left: govuk-spacing(6);
       }
 
       &:before {
         content: "\201C";
         float: left;
         clear: both;
-        margin-left: -$gutter-half;
+        margin-left: -(govuk-spacing(3));
```

Comment:
On line 35 the negative margin is done in this way: `margin: 0 0 0 (- govuk-spacing(6));` but here it's done differently. Do both work? Should we make them consistent?

Reply:
😄 I've just updated them from `(- govuk-spacing(3))` to `(govuk-spacing(3)` to address Jani's comments https://github.com/alphagov/govuk_publishing_components/pull/932#discussion_r294375918.

They both work in single attribute declaration. See my comment on
https://github.com/alphagov/govuk_publishing_components/pull/932#discussion_r294385794